doc/communication_team.rst

```diff
@@ -11,6 +11,6 @@
     </div>
     <div>
     <a href='https://github.com/francoisgoupil'><img src='https://avatars.githubusercontent.com/u/98105626?v=4' class='avatar' /></a> <br />
-    <p>francoisgoupil</p>
+    <p>François Goupil</p>
```

Comment:
I assume that Francois changed is name linked to his profile. It should be fine to update it at the same time.